tests/foreman/api/test_repository.py

```diff
@@ -676,6 +676,30 @@ def test_positive_upload_contents_srpm(self):
 
     @run_only_on('sat')
     @tier1
+    def test_positive_remove_contents(self):
+        """Synchronize a repository and remove rpm content.
+
+        @id: f686b74b-7ee9-4806-b999-bc05ffe61a9d
+
+        @Assert: The repository's content is removed and content counts shows
+        zero packages
+        """
+        # Create repository and synchronize it
+        repo = entities.Repository(
+            url=FAKE_2_YUM_REPO,
+            content_type='yum',
+            product=self.product,
+        ).create()
+        repo.sync()
+        self.assertGreaterEqual(repo.read().content_counts['rpm'], 1)
+        # Find repo packages and remove them
+        packages = entities.Package(repository=repo).search(
```

Comment:
Actually, there are more than one package, 32 if I'm not wrong, so it is required to pass all the ids to remove all packages.
I've used `assertGreaterEqual` and `1` for several reasons:
1. There is a little chance but still it is possible that number of packages at that repo or perhaps repo itself will be changed in the future. That's why not `assertEqual` and exact number of packages were used.
2. In case of `assertGreater` and `0` it is hard to see the difference between this and `assertEqual` and `0`, while `1` catches reviewer's eye.